jsk_perception/node_scripts/label_image_decomposer.py

```diff
@@ -78,6 +78,7 @@ def __init__(self):
         self.pub_img = self.advertise('~output', Image, queue_size=5)
         self.pub_label_viz = self.advertise('~output/label_viz', Image,
                                             queue_size=5)
+        self._only_label = rospy.get_param('~only_label', False)
```

Comment:
@wkentaro I'd recommend to update docs also about this param.

Reply:
Thanks. I updated.